### PR TITLE
Explain the pay as you go model for hosted zones and DNS query volume.

### PR DESCRIPTION
**Pull request by nestorsalceda**
This PR:

- Adds the reference to the pay-as-you-go model for hosted zones and DNS query volume.
- Requires merging of https://github.com/dnsimple/dnsimple-support/pull/1085

Belongs to https://github.com/dnsimple/dnsimple-business/issues/1741

### CODE REVIEW

**Review by ggalmazor — CHANGES_REQUESTED** [review]
I think it's important to highlight that it's about the activated zones. “Hosted” might be confused with something else  (as in registered vs hosted domains)

File: content/articles/dnsimple-plans.markdown [hunk 1]
```diff
@@ -17,7 +17,7 @@ You'll find all [the features listed for each plan on our pricing page](https://
 
 ## Solo
 
-No base subscription fee.
+No base subscription fee. You will be charged on a pay-as-you-go basis, with costs determined by the number of hosted zones and the volume of DNS queries each zone handles.
```
[the remark this answers is not in the document]
Comment (nestorsalceda) [reply content/articles/dnsimple-plans.markdown:20]: I think the active concept is important. I do agree with you. Thank you!

**Review by ggalmazor — APPROVED** [review]

File: content/articles/dnsimple-plans.markdown [hunk 2]
```diff
@@ -17,7 +17,7 @@ You'll find all [the features listed for each plan on our pricing page](https://
 
 ## Solo
 
-No base subscription fee.
+No base subscription fee. You will be charged on a pay-as-you-go basis, with fees determined by the number of active zones and their DNS query volume.
```
Comment (itsalyse) [inline content/articles/dnsimple-plans.markdown:20]: The only thing I would add here (once it's finished) is a link to the query volume & fees page

Comment (nestorsalceda) [reply content/articles/dnsimple-plans.markdown]: Updated and re-requested the review.

Thank you very much @itsalyse !!

**Review by itsalyse — APPROVED** [review]

**Pull request merged**